jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/InflaterTest.java

```diff
@@ -27,9 +27,16 @@
 import java.util.zip.DeflaterOutputStream;
 import java.util.zip.Inflater;
 import java.util.zip.ZipException;
+//import libcore.junit.junit3.TestCaseWithRules;
```

Comment:
Follow the convention commenting out lines. Same below for the rest of the file.

Reply:
added J2ObjC comments